packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeaderCell.tsx

```diff
@@ -15,19 +30,66 @@ export const DatepickerCalendarHeaderCell = compose<
   'span',
   DatepickerCalendarHeaderCellProps,
   DatepickerCalendarHeaderCellStylesProps,
-  BoxProps,
+  {},
   {}
->(Box, {
-  className: datepickerCalendarHeaderCellClassName,
-  displayName: 'DatepickerCalendarHeaderCell',
-  overrideStyles: true,
-  shorthandConfig: {
-    mappedProp: 'content',
+>(
+  (props, ref, composeOptions) => {
+    const context = useFluentContext();
+    const { setStart, setEnd } = useTelemetry(composeOptions.displayName, context.telemetry);
+    const { className, design, styles, variables, content } = props;
+    setStart();
+
+    const unhandledProps = useUnhandledProps(composeOptions.handledProps, props);
+    const ElementType = getElementType(props);
+    const getA11yProps = useAccessibility(props.accessibility, {
+      debugName: composeOptions.displayName,
+      mapPropsToBehavior: () => ({}),
+      rtl: context.rtl,
+    });
+
+    const { classes } = useStyles<DatepickerCalendarHeaderCellStylesProps>(DatepickerCalendarHeaderCell.displayName, {
+      className: composeOptions.className,
+      mapPropsToStyles: () => ({}),
+      mapPropsToInlineStyles: () => ({
+        className,
+        design,
+        styles,
+        variables,
+      }),
+      rtl: context.rtl,
+      composeOptions,
+      unstable_props: props,
+    });
+
+    const element = (
+      <ElementType
+        {...getA11yProps('root', {
+          className: classes.root,
+          ref,
+          ...unhandledProps,
+        })}
+      >
+        {createShorthand(Text, content, {
```

Comment:
In this case the content should be shorthand of the `TextProps`. Why exactly did we decide to go with this and change it to not just compose the `Box` component?

Reply:
This is the context for when the IE11 gridding was still in scope:
"I am playing with the IE11 gridding. 
 
The [DatepickerCalendarHeaderCell ](https://github.com/microsoft/fluentui/blob/042ed4c59b8d9547aaa40696ab6938a09def177e/packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeaderCell.tsx#L14)is [part ](https://github.com/microsoft/fluentui/blob/042ed4c59b8d9547aaa40696ab6938a09def177e/packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx#L290)of Grid setup. I read the [description ](https://fluentsite.z22.web.core.windows.net/components/box/definition)of Box component here. I've been then wondering whether I should change the component of headerCell to Text."

The answer was that I can change it. 

Since we no longer need to do IE11 gridding, should I revert?

Reply:
Yeah, let's return `Box` compose back :)

Reply:
Reverted.